src/NuGet.Core/NuGet.Configuration/PackageSource/PackageSource.cs

```diff
@@ -145,7 +145,7 @@ public bool Equals(PackageSource other)
                 return false;
             }
 
-            return Name.Equals(other.Name, StringComparison.CurrentCultureIgnoreCase) &&
```

Comment:
After this change search both `encyclopaedia and encyclopædia sources` as separate sources.

Reply:
Were they treated the same before this change?

Reply:
Reverting because  it's already doing `Source` compare with `StringComparison.OrdinalIgnoreCase`. Here actually `Source` is more important than `Name`.